### ISSUE TITLE
Vimeo90K Matlab Version SSIM

### ISSUE THREAD

**Issue by hzwer** [sniklaus/softmax-splatting#24]
Hi, Sniklaus!
    Would you mind following this script to provide SoftSplat's SSIM test results? CAIN, DAIN and some other papers use this test method. https://github.com/hzwer/arXiv2020-RIFE/blob/main/Vimeo90K_benchmark.py. 
    If you can give the result of UCF101 OTHER set to me or report the IE, I will be very grateful.
    Thanks again.

**Comment by sniklaus** [comment]
I am using the following, and am not aware whether one way of computing the SSIM is better than the other.

https://github.com/sniklaus/softmax-splatting/blob/713210d82f3eae5d7cdd29d75e6f16d0fd44fab2/benchmark.py#L106

If you like to compute different metrics, feel free to download the interpolation results from: http://sniklaus.com/softsplat-results

For your convenience, you can use the following commands to download them all one by one.

- Vimeo-90k: https://paste.ee/p/7WpSv
- UCF101 - DVF: https://paste.ee/p/Iwikc
- Middlebury: https://paste.ee/p/vqtpz

**Comment by francois-rozet** [comment]
Hello @sniklaus @hzwer, I've been working on benchmarking current image interpolation methods for the past few months (for an internship). I realized that a lot of researchers use different implementations of the same metrics for their evaluation. So I made a Python package (piqa: https://github.com/francois-rozet/piqa) to ease the work of everyone. Check it out if you are interested ;)

I've tested the metrics against the original (or most used) implementations to be sure mine are consistent.
